receiver/cloudfoundryreceiver/converter.go

```diff
@@ -41,15 +46,31 @@ func convertEnvelopeToMetrics(envelope *loggregator_v2.Envelope, metricSlice pme
 	}
 }
 
+func convertEnvelopeToLogs(envelope *loggregator_v2.Envelope, logSlice plog.LogRecordSlice, startTime time.Time) error {
+	log := logSlice.AppendEmpty()
+	log.SetTimestamp(pcommon.Timestamp(envelope.GetTimestamp()))
+	log.SetObservedTimestamp(pcommon.NewTimestampFromTime(startTime))
+	logLine := string(envelope.GetLog().GetPayload())
+	log.Body().SetStr(logLine)
```

Comment:
how about adding a `//exhaustive:enforce` here, in case the envelope log has a new log type except for `Log_OUT `and `Log_ERR`, and you can also add a default case, return a error called "unsupported log type"